shared/src/iosMain/kotlin/com/mountainspotter/shared/ai/IOSTensorFlowLiteModels.kt

```diff
@@ -0,0 +1,284 @@
+package com.mountainspotter.shared.ai
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import platform.CoreML.*
+import platform.Foundation.*
+import platform.CoreVideo.*
+import platform.CoreGraphics.*
+import platform.UIKit.*
+import kotlin.math.*
+
+/**
+ * iOS-specific CoreML model implementation
+ */
+class IOSTensorFlowLiteModel : TensorFlowLiteModelInterface {
```

Comment:
please ensure this and other classes compiles for ios using `gradle compileKotlinIosArm64` its currently not